solidity/contracts/BeaconRewards.sol

```diff
@@ -0,0 +1,86 @@
+pragma solidity ^0.5.17;
+
+import "./Rewards.sol";
+import "./KeepRandomBeaconOperator.sol";
+import "./TokenStaking.sol";
+
+contract BeaconRewards is Rewards {
+    KeepRandomBeaconOperator operatorContract;
+    TokenStaking tokenStaking;
+    uint256 constant _minimumKeepsPerInterval = 2; // TODO define
+
+    constructor (
+        address _token,
+        uint256 _firstIntervalStart,
+        uint256[] memory _intervalWeights,
+        address _operatorContract,
+        address _stakingContract
+    ) public Rewards(
+        _token,
+        _firstIntervalStart,
+        _intervalWeights
+    ) {
+        operatorContract = KeepRandomBeaconOperator(_operatorContract);
+        tokenStaking = TokenStaking(_stakingContract);
+    }
+
```

Comment:
Thoughts on adding:
```
    function receiveReward(uint256 i) public {
        receiveReward(bytes32(i));
    }
``` 
here and to `BeaconBackportRewards`? This would make web3 calls easier.